sktm/jenkins.py

```diff
@@ -320,6 +326,56 @@ def build(self, baserepo=None, ref=None, baseconfig=None,
         logging.info("submitted build: %s", build)
         return build.get_number()
 
+    def __get_job(self):
+        """
+        Get Jenkens job by job name. Retry Jenkins N times in case of temporary
```

Comment:
Can you mention the `retry_count` and the default value here instead of `N` so it's clear where the value comes from?